supplementary_style_guide/style_guidelines/mananged-services.adoc

```diff
@@ -0,0 +1,107 @@
+[[cloud-services]]
+= Cloud services
+
+Cloud services, also referred to as cloud services, are applications that run on a platform backed by data storage infrastructure and are hosted and managed by Red Hat on behalf of the customer. Managed cloud services provide rapid iteration and can more quickly deliver updated product functionality.
+
+This means a product is no longer running exclusively in the customer’s infrastructure but is a service running in the cloud that is operated and maintained by Red Hat.
+
+Red Hat is now responsible for ensuring the service is secure, scalable, performant, and available at all times.
+
+Cloud services documentation requires a shift from how documentation is typically created for an on-premise product. There is more emphasis on the content delivered within the user interface itself, a need to deliver content rapidly without necessarily having a set release cadence, and to adapt content to the needs of cloud service users. For example, there might not be a need for upgrade and migration content that’s typical for on-premise products.
+
+This section provides guidelines for creating content for the user interface, emphasizing accessibility. with the intent to provide consistency in how users understand and navigate across different cloud services.
+
+[[accessibility]]
+== Accessibility
+
+Ensure product documentation meets Section 508 compliance for accessibility. Accessibility standards are derived from the link:https://www.w3.org/TR/WCAG20/[Web Content Accessibility Guidelines (WCAGA)] and the link:https://www.access-board.gov/ict/[ICT Accessibility 508 Standards and 255 Guidelines].
+
+=== Images
+
+*Accessibility criteria:* The document provides text equivalents for every diagram, image, or other non-text element. Avoid using images of text instead of actual text.
+
+Evaluation:
+
+* All icons, images, diagrams, and non-text elements that convey information must have meaningful alternative-text descriptions.
+** For icons, use alternative text that describes what the icon does rather than what it looks like. For example, the alt text for a + icon is Add rather than Plus Sign.
+* The document is free from images of text; for example, a screen capture image of an informational table. Make sure that actual text is used to convey information, rather than images of text.
+
+=== Links and hypertext
+
+*Accessibility criteria:* The purpose of each link can be determined from the link text alone, or from the link text together with the link context. Links lead to the correct target locations within the document, or to a valid external Web page location (no broken links).
+
+* All link URLs and cross-reference links must provide descriptive text that conveys information about the content of the target linked page prior to clicking the link. Do not use a generic phrase like “click here.”
```

Comment:
Can you put the ending period outside of the double quotes? (New ISG rule)

Reply:
👍